### PR TITLE
PHP Dashboard Warnings and update about.php

### PR DESCRIPTION
**Pull request by mattyrob**
## Description
This PR contains the following:
- Adds more information about upstream security changes to `wp-admin/about.php`
- Fixes some admin PHP Warnings

## Motivation and context
Preparation for 1.5.0 RC

## How has this been tested?
Manual testing of Admin screens

### Before
Screenshot 2022-11-08 at 18 10 40
Screenshot 2022-11-08 at 18 10 58

### After
Screenshot 2022-11-08 at 18 11 19
Screenshot 2022-11-08 at 18 11 11

## Types of changes
- Bug fix

### CODE REVIEW

File: src/wp-admin/includes/nav-menu.php [hunk 1]
```diff
@@ -337,6 +337,7 @@ function wp_nav_menu_item_post_type_meta_box( $object, $box ) {
 	global $_nav_menu_placeholder, $nav_menu_selected_id;
 
 	$post_type_name = $box['args']->name;
```
Comment (xxsimoxx) [inline src/wp-admin/includes/nav-menu.php:339]: After this line seems that
`$post_type      = get_post_type_object( $post_type_name );`
is missing and causing some warnings.

**Review by xxsimoxx — CHANGES_REQUESTED** [review]

File: src/wp-admin/includes/nav-menu.php (hunk 1 above)
Comment (mattyrob) [reply src/wp-admin/includes/nav-menu.php:339]: It looks like we might need [47209](https://core.trac.wordpress.org/changeset/47209) here.

**Review by xxsimoxx — APPROVED** [review]
LGTM.

**Comment by mattyrob** [comment]
Thanks for testing this @xxsimoxx

**Pull request merged**